bamboo/create-release.sh

```diff
@@ -23,6 +23,7 @@ echo $RELEASE_URL
 curl -X POST -H "Authorization: token $GITHUB_TOKEN" --data-binary "@terraform-aws-cumulus.zip" -H "Content-type: application/octet-stream" $RELEASE_URL/assets?name=terraform-aws-cumulus.zip
 curl -X POST -H "Authorization: token $GITHUB_TOKEN" --data-binary "@terraform-aws-cumulus-workflow.zip" -H "Content-type: application/octet-stream" $RELEASE_URL/assets?name=terraform-aws-cumulus-workflow.zip
 curl -X POST -H "Authorization: token $GITHUB_TOKEN" --data-binary "@terraform-aws-cumulus-s3-replicator.zip" -H "Content-type: application/octet-stream" $RELEASE_URL/assets?name=terraform-aws-cumulus-s3-replicator.zip
+curl -X POST -H "Authorization: token $GITHUB_TOKEN" --data-binary "@terraform-aws-tea-distribution.zip" -H "Content-type: application/octet-stream" $RELEASE_URL/assets?name=terraform-aws-tea-distribution.zip
```

Comment:
Haven't figured out how to test this yet. It looks like it posts the .zip to GitHub. Presumably that happens on merge to master?

Reply:
No, this happens on release.   To test prior to release you need to do something like manually create the zip and run a deployment targeting a s3 zipfile.